boards/teensy40/src/main.rs

```diff
@@ -162,8 +162,18 @@ pub unsafe fn main() {
         peripherals.ccm.perclk_divider(),
     );
 
+    peripherals.dma.clock().enable();
+    peripherals.dma.reset_tcds();
+    peripherals
+        .lpuart2
+        .set_rx_dma_channel(&peripherals.dma.channels[7]);
```

Comment:
Are the DMA channels arbitrary or mapped to specific peripherals? `7` here and `8` below feel a bit magic-numbery, which might be fine, but if they could be replaced with a meaningful enum or something that might help readability.

Reply:
Mhmm, I think coalescing interrupts is probably a premature optimization at this stage. Capturing some of this design tradeoff in comments would be nice, but otherwise leaving things as they are should be fine for now.